src/security/token/TKTokenCryptoEngine.m

```diff
@@ -48,15 +53,75 @@ - (TKSignature *)signData:(NSData *)data
             usingKeyLevel:(Key_Level)keyLevel
                    reason:(NSString *)reason
                   onError:(OnError)onError {
+    if (!useLocalAuthentication) {
+        return [self createSignature:data usingKeyLevel:keyLevel];
+    }
+    
+    if (keyLevel >= Key_Level_Low) {
+        // We don't check DeviceOwnerAuthentication for Key_Level_Low
```

Comment:
The if statement is the opposite of the comment, right? We skip the "local" for anything...